repos/spack_repo/builtin/packages/icu4c/Quote_datagen.patch

```diff
@@ -0,0 +1,184 @@
+diff --git a/source/data/makedata.mak b/source/data/makedata.mak
+index 1fe8c88593c..36dcd4cd7b8 100644
+--- a/source/data/makedata.mak
++++ b/source/data/makedata.mak
+@@ -285,7 +285,7 @@ ICU_DATA_BUILD_VERBOSE=
+ # Keep track of whether they are built via timestamp files.
+ 
+ $(TOOLS_TS): "$(ICUTOOLS)\genrb\$(CFGTOOLS)\genrb.exe" "$(ICUTOOLS)\gencnval\$(CFGTOOLS)\gencnval.exe" "$(ICUTOOLS)\gencfu\$(CFGTOOLS)\gencfu.exe" "$(ICUTOOLS)\icupkg\$(CFGTOOLS)\icupkg.exe" "$(ICUTOOLS)\makeconv\$(CFGTOOLS)\makeconv.exe" "$(ICUPBIN)\pkgdata.exe"
+-	@echo "timestamp" > $(TOOLS_TS)
++	@echo "timestamp" > "$(TOOLS_TS)"
```

Comment:
(question) Based on the description this sounds like something that would also break outside of spack - is this something that should be upstreamed?

Also, do you think it would be worthwhile at all to encode this as a `FilterFile` since it seems to follow a pattern: e.g. wrap everything like `$(.*)` in quotes.

Reply:
Yes, this is non Spack specific.
Qt6 switched to a CMake based build and I don't believe they're still updating their buildsystem for the 5.x release series. That said, I'll look into it. If they're still accepting contributions I agree this would be valuable to upstream.